hack/configapi/main.go

```diff
@@ -70,6 +72,15 @@ func main() {
 			}
 		},
 	}
+	myCmd.PreRunE = func(cmd *cobra.Command, args []string) error {
+		if _, present := os.LookupEnv(envAwsKey); !present {
+			return fmt.Errorf("%s not set", envAwsKey)
+		}
+		if _, present := os.LookupEnv(envAwsKeyID); !present {
+			return fmt.Errorf("%s not set", envAwsKeyID)
+		}
+		return nil
+	}
```

Comment:
I am very surprised that this is needed.  We have multiple go tools that instantiate various aws sdk clients without explicitly needing environment variables.
The env vars can always be used as one of several authentication mechanisms.
I am logged in with the aws cli and never need to set those, the CI uses openid for login so I don't hink a check for those environment variables can be enforced.